src/domain/useCases/PermissionCase.ts

```diff
@@ -0,0 +1,43 @@
+import { ProcessPermissions } from '../entities/ProcessPermissions';
```

Comment:
Name of the file: prefix "Case"? "UseCase"? Or Processor?

I am not familiar with the term "Processor".